activesupport/lib/active_support/cache/redis_cache_store.rb

```diff
@@ -47,9 +47,11 @@ class RedisCacheStore < Store
         reconnect_attempts: 0,
       }
 
-      DEFAULT_ERROR_HANDLER = -> (method:, returning:, exception:) {
-        logger.error { "RedisCacheStore: #{method} failed, returned #{returning.inspect}: #{e.class}: #{e.message}" } if logger
-      }
+      DEFAULT_ERROR_HANDLER = -> (method:, returning:, exception:) do
+        if logger
+          logger.error { "#{name}: #{method} failed, returned #{returning.inspect}: #{exception.class}: #{exception.message}" }
```

Comment:
Using `name` means we'll have a long `ActiveSupport::Cache::RedisCacheStore:` message prefix instead of just `RedisCacheStore:`.

Reply:
Sure I'll fix this!